### PR TITLE
Add scripts for listing prs, reviews, and merging prs

### PR DESCRIPTION
**Pull request by ganemone**

### CODE REVIEW

File: src/repos.mjs [hunk 1]
```diff
@@ -1,108 +1,13 @@
...
-  {
-    upstream: 'fusionjs',
-    name: 'fusion-plugin-service-worker',
-  },
-];
+import api from './api';
 
-export default repos;
+export default async function getRepos() {
+  const {data} = await api.repos.getForOrg({
```
Comment (KevinGrandon) [inline src/repos.mjs:4]: Definitely cleaner, but we will now be iterating over stuff like the `rfcs` repos. I think ultimately we might want some sort of listing + tagging mechanism, so we can tag groups of repos to run scripts on.

File: README.md [hunk 2]
```diff
@@ -15,10 +15,18 @@ Runs setup and cleans local folders.
 
 ## Usage
 
-Requires node 9.1 or above.
+Requires node 8.9.1 or above.
```
Comment (KevinGrandon) [inline README.md:18]: Didn't realize that modules worked in 8.9, neat.

**Review by KevinGrandon — APPROVED** [review]
Awesome!

File: src/repos.mjs (hunk 1 above)
Comment (ganemone) [reply src/repos.mjs:4]: good point! I think in this case we can keep a blacklist rather than a whitelist

**Pull request merged**